### PR TITLE
[SIDE-478] Add convenience launcher functions

### PR DESCRIPTION
**Pull request by richseviora**
This PR adds two convenience launcher functions so the consumer doesn't need to explicitly instantiate and then call `serve` on the plugin object.

### CODE REVIEW

**Comment by iangallOlive** [comment]
I think these are great, but I think it would be nice if the func names were consistent across the Go and Node.js LDKs. I don't care which naming we use, but in the Go LDK, the functions are called `ServeControllerPlugin` and `ServeSensorPlugin`. They don't have to be consistent, so feel free to disagree :)

File: readme.md [hunk 1]
```diff
@@ -26,6 +26,27 @@ npm i
 ```
 
 ## Usage
+
+To launch a Controller:
+
+```javascript
+import { launchControllerPlugin } from 'ldk';
+import Controller from './controller'; // Your controller class.
```
Comment (iangallOlive) [inline readme.md:34]: Maybe these comments could say "Your implementation of the Controller interface" or something? It's probably fine how you have it, I'm being pedantic

Comment (richseviora) [reply readme.md:34]: Maybe "Your controller constructor"? :)  Just thinking that in this case we're instantiating it. Though the controller object doesn't need to be instantiated as a class, they could always roll a POJO.

**Comment by richseviora** [comment]
> hink these are great, but I think it would be nice if the func names were consistent across the Go and Node.js LDKs. I don't care which naming we use, but in the Go LDK, the functions are called `ServeControllerPlugin` and `ServeSensorPlugin`. They don't have to be consistent, so feel free to disagree :)

One of the hats I wore in my pre-developer life was developing a Knowledge Management Tool for a large Canadian telco. So I learned really quickly the benefit of being super consistent in your language, so you can get basically get me to agree to anything under the guise of "consistent language" :D I'll make the change

**Pull request merged**